_rules/descriptive-heading-b49b2e.md

```diff
@@ -98,22 +107,23 @@ Heading marked up with `h` element that is a single character that describes the
 </dl>
 ```
 
-#### Passed Example 5
+#### Passed Example 6
 
-Heading marked up with `role="heading"` that describes the topic or purpose of its section of the content. The heading is positioned off screen and is included in the Accessibility Tree.
+Heading marked up with `role="heading"` that describes the topic or purpose of its section of the content. The heading is positioned off screen and is included in the accessibility tree.
 
 ```html
-<span role="heading" style="position: absolute; top: -9999px; left: -9999px;"
+<span role="heading" aria-level="1"
+	style="position: absolute; top: -9999px; left: -9999px;"
 	>Opening Hours</span
 >
```

Comment:
Can we just not have the return 

````html
<span role="heading" aria-level="1"
	style="position: absolute; top: -9999px; left: -9999px;"
	>Opening Hours</span
  >

````html
<span role="heading" aria-level="1" style="position: absolute; top: -9999px; left: -9999px;">Opening Hours</span>